core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -307,7 +308,7 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
 
     // Validate testUser counts
     val result = runAndWait(workflowQuery.countWorkflowsByQueueStatusByUser)
-    result should contain key (testUserEmail)
+    result should contain key testUserId
```

Comment:
This looks like an API change to me. Granted it's admin endpoint but it was touched by A-Team in May so they probably care. Have they been consulted?